ci-operator/step-registry/openshift-qe/evpn/openshift-qe-evpn-commands.sh

```
@@ -0,0 +1,141 @@
+
+# 5. Run external FRR/VRF setup on bastion
+SSH_ARGS="-i ${CLUSTER_PROFILE_DIR}/jh_priv_ssh_key -oStrictHostKeyChecking=no -oUserKnownHostsFile=/dev/null"
+jumphost=$(cat "${CLUSTER_PROFILE_DIR}/address")
+bastion=$(cat "${CLUSTER_PROFILE_DIR}/bastion" 2>/dev/null || cat "${SHARED_DIR}/bastion")
+KUBECONFIG_PATH=$(cat "${CLUSTER_PROFILE_DIR}/kubeconfig")
+SETUP_SCRIPT_URL="https://raw.githubusercontent.com/kube-burner/kube-burner-ocp/main/cmd/config/scripts/setup_external_frr_vrf.sh"
+
+curl -fsSL -o /tmp/setup_external_frr_vrf.sh "${SETUP_SCRIPT_URL}"
+chmod +x /tmp/setup_external_frr_vrf.sh
+
+scp ${SSH_ARGS} -o ProxyCommand="ssh ${SSH_ARGS} -W %h:%p root@${jumphost}" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)